tests/Integration/Request/data/orderRetrieveTransactionsByCard.json

```diff
@@ -18,10 +18,10 @@
                     "card3DsStatusDescription": "MessageVersion=2.1.0,browser flow,3DS method not available,dynamic authentication,no cancel indicator,no status reason",
                     "cardBinCountry": "PL",
                     "firstTransactionId": "MCC0111LL1121"
-                },
-                "cardInstallmentProposal": {
-                    "proposalId": "5aff3ba8-0c37-4da1-ba4a-4ff24bcc2eed"
```

Comment:
according to the documentation https://developers.payu.com/europe/pl/api/#tag/Order/operation/retrieve-a-transaction `cardInstallmentProposal` is not a part of "card"